source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/organizations/settings/navigation/index.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from 'react';
+import { match as Match, withRouter, RouteComponentProps } from 'react-router';
+import { Menu } from 'semantic-ui-react';
+import { NavLink } from 'react-router-dom';
+
+import {
+  infoPath, userPath, productsPath, groupsPath, infrastructurePath
+} from '../routes';
+
+export interface Params {
+  orgId: string;
+}
+
+export type IProps =
+  & { match: Match<Params> }
+  & RouteComponentProps<{}>
+
+
+class Navigation extends React.Component<IProps> {
+  render() {
+    const { match } = this.props;
+    const { params: { orgId } } = match;
+
+    return (
+      <Menu vertical>
+
+        <Menu.Item exact as={NavLink}
+          to={infoPath.replace(/:orgId/, orgId)}
+          activeClassName='active'>
+          Basic Info
+        </Menu.Item>
+
+        {/* to={userPath.replace(/:orgId/, orgId)} */}
+        {/* as={NavLink} */}
+        {/* activeClassName='active' */}
```

Comment:
disabled for now, because I don't know how we're going to manage users per-org yet. I need to check if that was in scope or not.... or if the desired behavior will differ from the designs (where it just specifies auth0 creds)